input/script_utils.go

```diff
@@ -1880,12 +1882,16 @@ func NewLocalCommitScriptTree(csvTimeout uint32,
 //
 // The revocation path is simply:
 //
+//	<local_delayedpubkey> OP_CHECKSIG
```

Comment:
Should be `OP_DROP`.

Reply:
This is not fixed yet, can be confusing for future development.